spec/lib/policy_actions/search_indexer_spec.rb

```diff
@@ -18,7 +59,8 @@
       link: policy.base_path,
       slug: policy.slug,
       indexable_content: "",
-      organisations: [],
+      organisations: organisations.map {|o| o["base_path"].gsub(%r{.*/([^/]+)$}, '\1') },
```

Comment:
I think these tests would be stronger if the values expected here were explicit, i.e. `organisations: ['organisation-1', 'organisation-2']`. That way the test covers that the regexp exatracting the slug does what we expect it to do (i.e. extract the slug), rather than that it matches the regexp in the code.